sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JsonUtils.scala

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This might be too much. I am willing to revert this back if anyone feels it is a bit odd.

I just made this just for matching it to `CSVUtils` that contains variants for logically same preprocessing performed on different data (e.g. `Iterator`, `RDD`, `Dataset`).